artifacts/config_files/sfall-mods.ini

```diff
@@ -15,13 +21,13 @@ Containers=1
 ; 16 - bright yellow
 ; 32 - dark yellow
 ; 64 - purple
-OutlineColor=32
+OutlineColor=16
```

Comment:
Heh, first thing I did was change to bright yellow as well. Simply because I was used to it and it looks "Fallout" to me, while the not so bright yellow is somehow... not really sticking out, dunno how to describe it.